chapters/develop-organize/projects/reading/README.md

```diff
@@ -4,54 +4,69 @@ Owner: Alex Apostolescu
+RD: ... when comparted to [drills](../../drills/reading/README.md).
 Projects are mainly used as content for [assignments](../../../use-deliver/assignments/reading/README.md).
-The main goal of projects is to provide intense challenges to learners, providing them an increase in their skills and knowledge.
+The main goal of projects is to present intense challenges to learners, and, therefore, increase their skills and knowledge.
 
 Typically, a project consists of a statement, support files and, where possible, validators / checkers.
-The statement presents the context, goals, resources, useful references.
+The statement presents the context, goals, resources and useful references.
 Support files are pieces of digital information (source code, templates, application support files) that form a starting point for the project.
-Validators / checkers are scripts or applications that automate the verification and grading of the solution.
+Validators / checkers are scripts or applications that automate the verification and grading of solutions.
+RD: Define a solution before hand. Mention that a project is aimed to be solved and the solver creates a solution, typically as a digital archive to be submitted.
 Learners read the project statement, define a plan for reaching the state goals, implement a solution by using support files and then validate the solution with the checkers.
 
 Each project should have a reference solution.
+RD: Argument why this is important.
+RD: Also, I would rephrase this ^ as "must" instead of "should".
 Generally, the reference solution to the project is to be kept in a private repository.
 
 #### Storing Projects
 
+RD: Mention before hand that a project is stored in a project directory in the content repository.
+
 The project directory comprises of:
 
 - **Statement**
 
   Serving as the central component of a project, the statement provides a comprehensive description for each task.
-  It can include an introduction, details about the evaluation process, and additional references.
-  Typically, this content is stored in Markdown or PDF format to accommodate media files.
+  RD: The word "task" is too sudden. You didn't mention the assignment is a composition of tasks.
+  Often, it includes an introduction, details about the evaluation process, and additional references.
+  This content is stored in Markdown format with media files.
 
-- **Skeleton Files**
+- **Support Files**
 
   These act as the initial foundation for the project, establishing a consistent framework for all solutions.
   The project's skeleton may range from fill-in-the-blanks outlines and templates to straightforward guidelines.
   While the provision of skeleton files isn't feasible for every case, particularly open-ended assignments, when possible, it lays the groundwork for automation.
 
 - **Utility Files**
 
+` RD: This is a new term. Should we use "support" or should we keep "utility"?
```

Comment:
Using `utility/` provides a means to separate generic scripts (checker, linter) from the actual working space. I think this is a good practice that does not hinder anyone from simply deleting `utility/` and moving everything to `support/`.